IntegrationTests/ContentBlockerTests.swift

```diff
@@ -26,6 +26,8 @@ class ContentBlockerTests: XCTestCase {
         static let iFrames = "http://localhost:8000/iframetrackers.html"
         static let resources = "http://localhost:8000/resourcetrackers.html"
         static let requests = "http://localhost:8000/requesttrackers.html"
+        static let imageSrc = "http://localhost:8000/image_src.html"
+        static let xhr = "http://localhost:8000/xhr.html"
```

Comment:
Nice!

Reply:
Great to see the integration tests up and running again. The skipOnboarding() method is broken though, can that get fixed as part of this PR too?

Reply:
Yep, will have a look.